---------------------
PatchSet 10317 
Date: 2006/05/12 15:54:56
Author: hno
Branch: SQUID_2_5
Tag: (none) 
Log:
Bug #1434: SQUIDHOSTNAMELEN issues

cosmetic cleanup to get rid of remaining SQUIDHOSTNAMELEN magics which
may cause issues for very long hostnames.

Members: 
	include/util.h:1.62->1.62.2.1 

Index: squid/include/util.h
===================================================================
RCS file: /cvsroot/squid/squid/include/util.h,v
retrieving revision 1.62
retrieving revision 1.62.2.1
diff -u -r1.62 -r1.62.2.1
--- squid/include/util.h	17 Oct 2001 01:36:07 -0000	1.62
+++ squid/include/util.h	12 May 2006 15:54:56 -0000	1.62.2.1
@@ -45,16 +45,7 @@
 #endif
 
 #if !defined(SQUIDHOSTNAMELEN)
-#include <sys/param.h>
-#ifndef _SQUID_NETDB_H_		/* need protection on NEXTSTEP */
-#define _SQUID_NETDB_H_
-#include <netdb.h>
-#endif
-#if !defined(MAXHOSTNAMELEN) || (MAXHOSTNAMELEN < 128)
-#define SQUIDHOSTNAMELEN 128
-#else
-#define SQUIDHOSTNAMELEN MAXHOSTNAMELEN
-#endif
+#define SQUIDHOSTNAMELEN 256
 #endif
 
 #if defined(_SQUID_FREEBSD_)
